traffic_portal/app/src/common/modules/table/jobs/TableJobsController.js

```diff
@@ -17,10 +17,134 @@
  * under the License.
  */
 
-var TableJobsController = function(jobs, $scope, $state, locationUtils) {
+var TableJobsController = function(tableName, jobs, $document, $scope, $state, $uibModal, locationUtils, jobService, messageModel, dateUtils) {
+
+	/**
+	 * Gets value to display a default tooltip.
+	 */
+	function defaultTooltip(params) {
+		return params.value;
+	}
+
+	function dateCellFormatter(params) {
+		return params.value.toUTCString();
+	}
+
+	columns = [
+		{
+			headerName: "Delivery Service",
+			field: "deliveryService",
+			hide: false
+		},
+		{
+			headerName: "Asset URL",
+			field: "assetUrl",
+			hide: false
+		},
+		{
+			headerName: "Parameters",
+			field: "parameters",
+			hide: false
+		},
+		{
+			headerName: "Start (UTC)",
+			field: "startTime",
+			hide: false,
+			filter: "agDateColumnFilter",
+			tooltip: dateCellFormatter,
+			valueFormatter: dateCellFormatter
+		},
+		{
+			headerName: "Expires (UTC)",
+			field: "expires",
+			hide: false,
+			filter: "agDateColumnFilter",
+			tooltip: dateCellFormatter,
+			valueFormatter: dateCellFormatter
+		},
+		{
+			headerName: "Created By",
+			field: "createdBy",
+			hide: false
+		}
+	];
+
+	/** All of the jobs - startTime fields converted to actual Dates and derived expires field from TTL */
+	$scope.jobs = jobs.map(
+		function(x) {
+			// need to convert this to a date object for ag-grid filter to work properly
+			x.startTime = new Date(x.startTime.replace("+00", "Z"));
+
+			// going to derive the expires date from start + TTL (hours). Format: TTL:24h
+			let ttl = parseInt(x.parameters.slice('TTL:'.length, x.parameters.length-1), 10);
+			x.expires = new Date(x.startTime.getTime() + ttl*3600*1000);
+		});
 
 	$scope.jobs = jobs;
 
+	$scope.quickSearch = '';
+
+	$scope.pageSize = 100;
+
+	/** Options, configuration, data and callbacks for the ag-grid table. */
+	$scope.gridOptions = {
+		columnDefs: columns,
+		enableCellTextSelection: true,
+		defaultColDef: {
+			filter: true,
+			sortable: true,
+			resizable: true,
+			tooltip: defaultTooltip
+		},
+		rowClassRules: {
+			'active-job': function(params) {
+				return new Date(params.data.expires) > new Date();
```

Comment:
expires is already a Date according to the `.map` you did before, so this construction should be redundant.